### PR TITLE
Test function toCamelCase with jest

### PR DESCRIPTION
**Pull request by SimpleCookie**

### CODE REVIEW

File: convert.js [hunk 1]
```diff
@@ -50,23 +50,10 @@ function getConvertedArray(arr) {
   return arr.map((obj) => convert(obj));
 }
 
-function toCamelCase(key) {
```
Comment (SimpleCookie) [inline convert.js:53]: The old toCamelCase function did not pass 2 of the 3 tests. I remade it in the way I would've solved it.
You don't have to use my solution, it is your choice :) 
I just want to give inspiration 😄

Comment (BluePi-Cloud) [reply convert.js:53]: This code is very concise, I like it

Comment (koshuang) [reply convert.js:53]: @SimpleCookie that's great! the code is pretty neat! Thank you for your effort to contribute the testing and the implementation :)

Comment (SimpleCookie) [reply convert.js:53]: You are welcome :)

**Pull request merged**